apps/vmq_server/src/vmq_queue.erl

```diff
@@ -786,6 +789,9 @@ add_session_(
         cleanup_on_disconnect := Clean
     } = Opts,
     BatchSize = maps:get(queue_to_session_batch_size, Opts, 100),
+    IgnoreMaxQueueSize =
```

Comment:
What does MaybeIgnoreMaxMsgSize do and why the `and` here?

Reply:
It is meant as a precaution that online the offline -> online transition can overwrite the max_online_messages and not any other usage of add_session_. If "MaybeIgnoreMaxMsgSize" it might ignore the size of the max online message queue. Thinking about it, the naming is odd because it could be misinterpreted as the maximum message size, while I meant the maximimum messages size of the queue. Let me know what you think of the switch and I will rename it to something better (if the naming caught your attention)

Reply:
AllowExtendedQueueSize or similar...and I think here's one of the places where a short comment would be helpful in the code.